src/CrowdedMod/Patches/CreateGameOptionsPatches.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using HarmonyLib;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace CrowdedMod.Patches
+{
+    internal static class CreateGameOptionsPatches
+    {
+        [HarmonyPatch(typeof(CreateOptionsPicker), nameof(CreateOptionsPicker.Start))]
+        public static class CreateOptionsPicker_Start // Credits to XtraCube (mostly)
+        {
+            public const byte maxPlayers = 127;
+            
+            public static void Postfix(CreateOptionsPicker __instance)
+            {
+                if (__instance.mode != SettingsMode.Host) return;
+                var offset = __instance.MaxPlayerButtons[1].transform.position.x - __instance.MaxPlayerButtons[0].transform.position.x;
+
+                #region MaxPlayers stuff
+                
+                List<SpriteRenderer> playerButtons = __instance.MaxPlayerButtons.ToList();
+                
+                SpriteRenderer plusButton = Object.Instantiate(playerButtons.Last(), playerButtons.Last().transform.parent);
+                plusButton.GetComponentInChildren<TextRenderer>().Text = "+";
```

Comment:
this TextRenderer does not instantiate causing cosmetic bugs 
![image](https://user-images.githubusercontent.com/46283010/112747080-c283b580-8fbb-11eb-8c6a-4f7d50614755.png)
It's all working, just this